src/Forms/HTMLEditor/HTMLEditorSanitiser.php

```diff
@@ -17,8 +18,19 @@
  */
 class HTMLEditorSanitiser
 {
+    use Configurable;
     use Injectable;
 
+    /**
+     * rel attribute to add to link elements which have a target attribute (usually "_blank")
+     * this is to done to prevent reverse tabnabbing - see https://www.owasp.org/index.php/Reverse_Tabnabbing
+     * noopener includes the behaviour we want, though some browsers don't yet support it and rely
+     * upon using noreferrer instead - see https://caniuse.com/rel-noopener for current browser compatibility
+     *
+     * @var string
+     */
+    private static $link_noopener_value = 'noopener, noreferrer';
```

Comment:
Any specific reason you've made this config? I feel like it's just best practice.

Reply:
People might want to remove the norefferer since it may have an effect on numbers in google analytics. noreferrer is only there because of Microsoft browsers not properly retrospecting noopener